sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +74,12 @@ internal class RadarApiClient(
     }
 
     interface RadarSendEventApiCallback {
-        fun onComplete(status: RadarStatus, res: JSONObject? = null, events: Array<RadarEvent>? = null)
+        fun onComplete(
+            status: RadarStatus,
+            res: JSONObject? = null,
+            event: RadarEvent? = null,
+            user: RadarUser? = null
```

Comment:
depending on this comment, we will keep or leave `user`.
https://github.com/radarlabs/radar-sdk-android/pull/276/files#r1124618546

Looking here it seems that we populate the local `user` off of the response. Given how the user object returned doesn't have any context (nested within `event`) and just has IDs, I am thinking we should remove it. Let me know your thoughts